dockernet/scripts/community-pool-staking/README.md

```diff
@@ -0,0 +1,20 @@
+## Community Pool Staking Integration Tests
+* Set `HOST_CHAINS` to `(DYDX)` in `config.sh`
+* Start dockernet
+```bash
+make start-docker
+```
```

Comment:
call setup.sh to create the connections between dydx and noble?  Or is this not needed for phase1/3

Reply:
not needed for phase 1/3!

Reply:
but good call that we should update the readme when we go through stage2